doc/spelling_word_list.txt

```diff
@@ -0,0 +1,16 @@
+admin
+backend
+cacheable
+config
+css
+hostname
+hostnames
```

Comment:
@ddeboer i guess proper english grammar would be "host name" should we keep as is?